src/Plugin/Field/FieldFormatter/AdvancedEntityReferenceFormatter.php

```diff
@@ -109,7 +158,7 @@ public static function addMoreSubmit(array $form, FormStateInterface $form_state
    * @param \Drupal\Core\Form\FormStateInterface $form_state
    *   The current state of the form.
    */
-  public static function removeSubmit(array $form, FormStateInterface $form_state) {
+  public static function removeDeltaSubmit(array $form, FormStateInterface $form_state) {
     $items = $form_state->get('delta_items');
     $form_state->set('delta_items', $items - 1);
```

Comment:
If `$items = 0` then this will set `delta_items` to `-1` which will lead to unexpected behaviors.
Better validate `$items >= 1` before setting `$items - 1`

Reply:
fixed